src/templates/landingPage.tsx

```diff
@@ -29,17 +31,66 @@ const testEvents = [
 
 type Props = {
   pageContext: {
-    locale: string
+    locale: "fi" | "en"
```

Comment:
We should use the Locale type everywhere now that it's defined in `types/strapi.ts`
```suggestion
    locale: Locale
```